Documentation/dev-guide/limit.md

```diff
@@ -0,0 +1,8 @@
+# Request size limit
+
+The size limit is 1MB. etcd is designed to handle small key value paris. Very large request might block other requests for a long time. This also applies to `Txn` requests. We might loosen the size in the future a little bit or make it configurable.
```

Comment:
etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made it configurable.